deb-get

```diff
@@ -2413,6 +2413,16 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_beyond-compare() {
+    if [ "${ACTION}" != "prettylist" ]; then
+        URL="https://www.scootersoftware.com/bcompare-${BASE_VER}amd64.deb"
+        VERSION_PUBLISHED=$(curl -s -S https://www.scootersoftware.com/download.php | grep -Eo "Current Version:.*, released" | cut --complement -c -23 | sed 's/\,\ build\ /\./;s/\, released/\_/;s/_//')
```

Comment:
Instead of doing `s/\, released/\_/;s/_//`, just do `s/\, released//`. And if you define `VERSION_PUBLISHED` before `URL`, you can define `URL` as `https://www.scootersoftware.com/bcompare-${VERSION_PUBLISHED}_amd64.deb`.